src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -51,7 +51,11 @@
 
     * :http-client-cipher-suites - A list of legal SSL cipher suites that may
         be used when https client requests are made."
-  {:ruby-load-path              (schema/both [schema/Str] (schema/pred vector?))
+  ;; NOTE: there is a bug in the version of schema we're using, which causes
+  ;; the order of things that you put into a `both` to be very important.
+  ;; The `vector?` pred here MUST come before the `[schema/Str]`.  For more info
+  ;; see https://github.com/Prismatic/schema/issues/68
```

Comment:
Should we create a ticket for upgrading to the latest schema?  We're several versions behind, and I think we have other projects that are already on newer versions.